app/javascript/lib/table-item-anchors.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener("DOMContentLoaded", function () {
+  [...document.querySelectorAll("a#back-button")].map(function (btn) {
+    btn.addEventListener("click", function (evt) {
+      evt.preventDefault();
+      history.back();
+    });
+    return btn;
+  });
+  [...document.querySelectorAll("table.govuk-table tr[id] a")].map(
```

Comment:
would prefer to be explicit about this I think... so
```
"table.store-anchor-on-click tr[id] a"
```

Reply:
Sure I can add more specificity